app/src/main/java/de/danoeh/antennapod/fragment/FeedInfoFragment.java

```diff
@@ -211,25 +226,50 @@ private void showFeed() {
 
         txtvDescription.setText(description);
 
-        if (!TextUtils.isEmpty(feed.getAuthor())) {
-            txtvAuthor.setText(feed.getAuthor());
-            txtvAuthorHeader.setText(feed.getAuthor());
-        } else {
-            lblAuthor.setVisibility(View.GONE);
-            txtvAuthor.setVisibility(View.GONE);
-        }
-        if (!TextUtils.isEmpty(feed.getLanguage())) {
-            txtvLanguage.setText(LangUtils.getLanguageString(feed.getLanguage()));
-        } else {
-            lblLanguage.setVisibility(View.GONE);
-            txtvLanguage.setVisibility(View.GONE);
-        }
         txtvUrl.setText(feed.getDownload_url() + " {fa-paperclip}");
         Iconify.addIcons(txtvUrl);
-
         getActivity().invalidateOptionsMenu();
     }
 
+    private void loadStatistics() {
+        if (disposable != null) {
+            disposable.dispose();
+        }
+
+        disposable =
```

Comment:
Please create a new variable for this (eg. `disposableStatistics`) and make sure to dispose in `onDestroy`, like it is done with the other one. Otherwise, loading statistics and loading feed details could interfere.